.env.example

```diff
@@ -0,0 +1,2 @@
+# On local environments this should be your computer local IP address
+API_URL = API IP or DNS url
```

Comment:
I suggest adding an example like `http://127.0.0.1:8080` so that you don't have to guess how it will be interpreted.